src/executables/util_cs.c

```diff
@@ -4291,6 +4279,20 @@ flashback (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  for (i = 0; i < num_tables; i++)
+    {
+      table_name = utility_get_option_string_value (arg_map, OPTION_STRING_TABLE, i + 1);
+
+      /* when table_name doesn't contain the user name (owner), then attach the user name to the table name */
+      sm_user_specified_name (table_name, table_name_buf, SM_MAX_IDENTIFIER_LENGTH);
```

Comment:
It should be checked that the length of table_name does not exceed the length of table_name_buf (SM_MAX_IDENTIFIER_LENGTH).

Reply:
Okay, I understand what you mean.
I will check a length of table_name.

Reply:
trivial;

Is there any reason that this for loop moves here?